ROMFS/px4fmu_common/init.d/50000_generic_ground_vehicle

```diff
@@ -0,0 +1,60 @@
+#!nsh
+#
+# @name Generic Ground Vehicle
+#
+# @type Rover
+# @class Rover
+#
+# @output MAIN2 steering
+# @output MAIN4 throttle
+#
+# @maintainer
+#
+
+sh /etc/init.d/rc.ugv_defaults
```

Comment:
Does this exist? I see rc.gnd_defaults ubt not ugv_default.

Reply:
Good catch, I pulled this from #7628, but it's since been renamed. Thanks